runconfig/parse.go

```diff
@@ -42,6 +42,9 @@ var (
 	ErrConflictNetworkExposePorts = fmt.Errorf("Conflicting options: --expose and the network mode (--net)")
 )
 
+// DefaultSHMSize is the default size of the SHM which will be mounted in the container
+const DefaultSHMSize int64 = 67108864
```

Comment:
I'd suggest to mention the `64MB` here (so that people don't have to calculate what this number is), and remove it in other places (will add some comments)